libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -23,46 +23,40 @@ using std::ostream;
 using std::ios;
 
 namespace{
-    bool enableDataPath = true;
-
-    //--------------------------------------------------
-    string defaultDataPath(){
-    #if defined TARGET_OSX
-        try{
-            return std::filesystem::canonical(ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../../../data/")).string();
-        }catch(...){
-            return ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../../../data/");
-        }
-    #elif defined TARGET_ANDROID
-        return string("sdcard/");
-    #else
-        try{
-            return std::filesystem::canonical(ofFilePath::join(ofFilePath::getCurrentExeDir(),  "data/")).make_preferred().string();
-        }catch(...){
```

Comment:
Thanks @dimitre! 
The only thing I am wondering about is if we are losing anything with the removal of the try{ catch in the old defaultDataPath().  Do you know if the new behavior is the same or could there be some breakage here?

Reply:
@ofTheo, thanks for reading, I've reverted back to try and catch and pushed a new commit.
I'm not sure why I changed that before.

Reply:
thanks @dimitre 
I think we need it for the #else section too ( last commit was just for `#if defined TARGET_OSX` 

Then this looks good to go! 👍
Thanks!!

Reply:
Great, done